src/utils/openai.ts

```diff
@@ -126,7 +127,7 @@ export const generateCommitMessage = async (
 				top_p: 1,
 				frequency_penalty: 0,
 				presence_penalty: 0,
-				max_tokens: 200,
+				max_tokens: length,
```

Comment:
As I understand, I believe this includes the length of our input, so we have to add it here.

https://help.openai.com/en/articles/5072518-controlling-the-length-of-completions

Reply:
Hmm this makes sense.

Reply:
Made the changes. Can please you have a look?

Reply:
Just a friendly ping :wave: @privatenumber